src/jats/importer/jats-dom-parser.ts

```diff
@@ -863,7 +863,13 @@ const nodes: NodeRule[] = [
     node: 'section',
     getAttrs: (node) => {
       const element = node as HTMLElement
-
+      const grandParentNodeName =  element.parentNode?.parentNode?.nodeName.toLowerCase()
+      if ( grandParentNodeName !== 'body' ) {
+        return {
+          id: element.getAttribute('id'),
+          category: 'MPSectionCategory:subsection',
+        }
+      }
```

Comment:
We can just set the sec-type attr to 'subsection', we don't need to explicitly return this.
 `element.setAttribute('sec-type', 'subsection')`

Reply:
thanks, another question is should we check if the grandParentNodeName is defined?  What if the node doesn't have a grand parent? The current implementation would set it as a subsection, no?

Reply:
yes, fixed.